velox/functions/prestosql/aggregates/ApproxDistinctAggregate.cpp

```diff
@@ -199,7 +199,11 @@ class ApproxDistinctAggregate : public exec::Aggregate {
         });
   }
 
-  void destroy(folly::Range<char**> /*groups*/) override {}
+  void destroy(folly::Range<char**> groups) override {
+    for (auto group : groups) {
+      value<HllAccumulator>(group)->~HllAccumulator();
```

Comment:
Do we need isNull(group) check here? CC: @Yuhta

Reply:
No we don't, the ctor is called unconditionally so the dtor should also.